docs/usage.rst

```diff
@@ -4,125 +4,87 @@ Usage
-``requests`` packages) with the following parameters (you can do this
-out of the OpenStack client it is just a call to the public API):
+   ``kubectl create -f reana-resources-k8s/deployments/reana-system``
 
-``python cli.py fibonacci -f fib_file -e alice``
+-  Yadage workers:
 
-Once you run the command you should start seeing that Kubernetes has
-launched some jobs.
+   ``kubectl create -f reana-resources-k8s/deployments/yadage-workers``
 
-::
+-  Services:
 
-    kubectl get jobs --all-namespaces
+   ``kubectl create -f reana-resources-k8s/services/``
 
-    NAMESPACE NAME DESIRED SUCCESSFUL AGE default
-    bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624-0 1 0 1m default
-    bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624-1 1 0 1m default
-    bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624-2 1 0 1m default
-    bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624-3 1 0 1m default
-    bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624-4 1 0 1m
+-  Secrets (you should provide your own CephFS secret):
 
-For checking the results you can connect to node the``\ storage-admin\`
-node which is hosting all the nodes:
+   ``kubectl create -f reana-resources-k8s/secrets/``
 
+Get the Workflow Controller ip address and port
+-----------------------------------------------
 ::
 
-    $ kubectl exec -ti storage-admin bash
-    ~ cd /mnt/ceph/alice
-    ~ ls -lrt``
-
-An example could be:
-::
+    $ kubectl describe services workflow-controller | grep NodePort
+    NodePort:                  http                    32313/TCP
+    $ kubectl get pods | grep workflow-controller | cut -d" " -f 1 | xargs kubectl describe pod | grep 'Node:'
+    Node:        192.168.99.100,192.168.99.100
 
-    /bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624/0/input.dat
-    /bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624/0/output.dat
+So the Workflow Controller component can be accessed through ``192.168.99.100:32313``.
 
-Where you can have a look on the input and output data for the step 0 of
-the workflow bdfdbb1d-e832-46e3-9bf6-6e1ab8d57624.
+Launch workflows
+----------------
 
 Launch Yadage workflows
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Enter the ``tests`` directory and call cli.py (requires ``click`` and
-``requests`` packages) with the following parameters (you can do this
-out of the OpenStack client it is just a call to the public API):
-
-``python cli.py yadage --experiment cms --toplevel some-text --workflow some-text --parameters '{"par1" : "val1"}'``
-
-::
-
-    $ kubectl get pods | grep cms-yadage-workflow-controller | cut -d" " -f1 | xargs kubectl logs
-    [2016-11-18 09:40:41,251: INFO/MainProcess] Received task: tasks.run_yadage_workflow[dddde31a-ce2f-46f8-b4dc-46a8a1e4f5f9]
-    [2016-11-18 09:40:41,256: WARNING/Worker-1] some-text
-    [2016-11-18 09:40:41,256: WARNING/Worker-1] some-text
-    [2016-11-18 09:40:41,256: WARNING/Worker-1] {"par1" : "val1"}
-    [2016-11-18 09:40:41,259: INFO/MainProcess] Task tasks.run_yadage_workflow[dddde31a-ce2f-46f8-b4dc-46a8a1e4f5f9] succeeded in 0.00235133804381s: None
+FIX ME
```

Comment:
Please use `FIXME` as one word.  The advantage is that `pyflake/pylint` report  FIXME lines, so one can `grep` for the same text everywhere.